### DISCUSSION TITLE
Redis通信协议

### DISCUSSION THREAD

**Discussion by giscus** [ichengzi/ichengzi.github.io#50] [Announcements]
# Redis通信协议

redis 的通信协议是基于字符串的，设计的很轻巧， 人工可读。

https://ichengzi.github.io/2021/10/01/redis%E9%80%9A%E4%BF%A1%E5%8D%8F%E8%AE%AE.html

**Comment by ichengzi** [comment]
redis 这个极简文本交互协议， 可以使用很多基础设施， 比如telnet
